lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -48,9 +48,9 @@ def serializable_hash(options = nil)
         document = if serializer.success?
                      success_document(options)
                    else
-                     failure_document
+                     failure_document(options)
```

Comment:
In the sense that if `:transform` is in those options that transform will be used instead of the global option or the adapter default (`:dashed`).